content/it/docs/reference/glossary/kube-proxy.md

```diff
@@ -0,0 +1,24 @@
+---
+title: kube-proxy
+id: kube-proxy
+date: 2018-04-12
+full_link: /docs/reference/command-line-tools-reference/kube-proxy/
+short_description: >
+  `kube-proxy` è un proxy eseguito su ogni nodo del cluster.
+
+aka:
+tags:
+- fundamental
+- networking
+---
+ [kube-proxy](/docs/reference/command-line-tools-reference/kube-proxy/) è un
+proxy eseguito su ogni nodo del cluster, che implementa parte dei
+Kubernetes {{< glossary_tooltip term_id="service">}}.
+
+<!--more-->
+
+I kube-proxy mantengono le regole di networking sui nodi. Queste regole permettono
+la comunicazione verso i tuoi da altri nodi del cluster o dall'esterno.
+
+Il kube-proxy usa l'infrastruttura di sistema di gestione pacchetti se è presente
```

Comment:
usa l'infrastruttura di sistema di gestione pacchetti se è presente ... --> usa le librerie del sistema operativo quando possible; in caso contrario ...